### PR TITLE
Org changes 190613

### PR DESCRIPTION
**Pull request by thereseanders**
- Adding in documentation changes in `org.R`
- Combining changes in `invite_user` between @rundel's and my version #35 (does not fix re-invitation issue but outputs separate messages for added users, existing members, and pending members)
- Deprecate functions and move to bottom of `org.R`
- Use `org` in user-facing functions and `owner` in functions interacting with API
- Re-run `rogygen2` documentation and `pkgdown` site building.

NOTE: Changes in `repo_files.R`, in particular changes relating to `add_file` in a separate PR.

### CODE REVIEW

**Comment by thereseanders** [comment]
Sorry, I just saw that this commit included changes to `add_file` that were not supposed to be included. Will open another PR.

File: R/org.R [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
 #' }
 #'
 #' @family github organization related functions
 #'
 #' @export
 #'
-get_repos = function(org, filter=NULL, exclude=FALSE, full_repo=TRUE) {
-  stopifnot(length(org)==1)
-  stopifnot(length(filter)<=1)
+get_repo = function(org, filter = NULL, exclude = FALSE, full_repo = TRUE) {
```
[the remark this answers is not in the document]
Comment (mine-cetinkaya-rundel) [reply R/org.R]: Now it's working!

File: R/org.R [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
+#' @param exclude Logical. Should entries matching the regular expression in `filter` be excluded or included?
+#' @param full_repo Logical. Should the full repository address be returned (e.g. `owner/name` instead of just `repo`)?
 #'
-#' @aliases get_repo
 #'
 #' @examples
 #' \dontrun{
-#' get_repos("ghclass")
-#' get_repos("ghclass", "hw1-")
+#' get_repo("ghclass")
```
[the remark this answers is not in the document]
Comment (mine-cetinkaya-rundel) [reply R/org.R]: We'll do this in a separate PR all at once.

File: R/org.R [hunk 3]
```diff
@@ -1,39 +1,38 @@
...
-#' \code{get_repos} returns a (filtered) vector of repos belonging to a GitHub organization.
+#' `get_repo` returns a (filtered) vector of repositories belonging to a GitHub organization.
 #'
-#' @param org character, name of the GitHub organization.
-#' @param filter character, regex pattern for matching (or excluding) repos.
-#' @param exclude logical, should entries matching the regex be excluded or included.
-#' @param full_repo logical, should the full repo name be returned (e.g. \code{org/repo} instead of just \code{repo})
+#' @param org Character. Name of the GitHub organization.
+#' @param filter Character. Regular expression pattern for matching (or excluding) repositories.
+#' @param exclude Logical. Should entries matching the regular expression in `filter` be excluded or included?
```
[the remark this answers is not in the document]
Comment (mine-cetinkaya-rundel) [reply R/org.R]: Will implement in a separate PR in one swoop

File: R/org.R [hunk 4]
```diff
@@ -43,137 +42,329 @@ get_repos = function(org, filter=NULL, exclude=FALSE, full_repo=TRUE) {
...
+
+  res = github_api_get_members(org)
+  members = purrr::map_chr(res$result, "login")
+
+  filter_results(members, filter, exclude)
+}
+
+github_api_get_invitations = function(org) {
+
+  .Deprecated(msg = "'github_api_get_invitations' will be removed in the next version. Use 'github_api_get_invitation' instead.",
```
[the remark this answers is not in the document]
Comment (mine-cetinkaya-rundel) [reply R/org.R]: Will do in a separate PR

**Comment by thereseanders** [comment]
https://github.com/rundel/ghclass/pull/36/commits/15f2e15b25d1d1fffd82313251e39ac3d98080ea fixes the remaining `invite_user` issues. The problem with the additional empty line was related to the `status_msg` function that per default out out the empty line with checkmark from `usethis::ui_done()`. 
- Change message narrative of `invite_user`.
- Fix `status_msg` output

**Pull request merged**